controllers/cluster_controller_phases.go

```diff
@@ -90,7 +90,7 @@ func (r *ClusterReconciler) reconcileExternal(ctx context.Context, cluster *clus
 
 	if !util.HasOwnerRef(obj.GetOwnerReferences(), ownerRef) {
 		obj.SetOwnerReferences(util.EnsureOwnerRef(obj.GetOwnerReferences(), ownerRef))
-		if err := r.Patch(ctx, obj, objPatch); err != nil {
+		if err := r.Client.Patch(ctx, obj, objPatch); err != nil {
```

Comment:
I think I remember a comment from the other day about doing this. If we're embedding, there is no need to make this change. If we want to require that it look like this, then we should move the embedded type to a proper field.

Reply:
I went with it to be more explicit and align with our other uses. It also helps with discoverability when searching the code for uses of Patch/Update :)

Reply:
I'm going to file an issue to make it a proper field.